tensorflow/python/kernel_tests/topk_op_test.py

```diff
@@ -208,6 +212,8 @@ def testKTooLarge(self):
       nn_ops.top_k(inputs, 4)
 
   @test_util.run_deprecated_v1
+  # scatter_nd is not implemented in DML.
+  @test_util.skip_dml
```

Comment:
Don't disable the test because scatter_nd isn't implemented, especially if you don't link a bug for it. We have many failing tests because of scatter_nd, so it would be a pain to hunt all the disabled tests. ScatterNd is also probably one of the next operators that I'm going to implement, so it won't fail for too long.